src/bin/agent/CommandLockAll.ts

```diff
@@ -58,7 +58,7 @@ class CommandLockAll extends CommandPolykey {
           logger: this.logger.getChild(WebSocketClient.name),
         });
         pkClient = await PolykeyClient.createPolykeyClient({
-          streamFactory: () => webSocketClient.startConnection(),
+          streamFactory: (ctx) => webSocketClient.startConnection(ctx),
```

Comment:
For the `streamFactory` we're passing a `ContextTimed`, the timer is used for timeout for the connection here since the websocket makes a new connection per stream. The signal is used to abort an active connection.

For the quic system only the signal will be used to abort an active stream. A timer could possibly be used to timeout the stream negotiation if the connection is not responding at this point but I'm unsure of the internal details here.

Reply:
See: https://github.com/MatrixAI/Polykey/pull/513#discussion_r1156712794